src/date.ts

```diff
@@ -65,5 +63,15 @@ export function makeRelativeDate(date: Date): string | null {
 }
 
 export function formatDate(date: Date): string {
-    return moment(date).format('HH:mm dddd, D MMMM YYYY');
+    const options = {
+        weekday: "long", year: "numeric", month: "long",
+        day: "numeric", hour: "numeric", minute: "numeric"
+    }
+    const dateParts = new Date(date)
+        .toLocaleTimeString("en-us", options)
```

Comment:
This doesn't quite feel right. Are you removing moment to save on the bundle size?
If so, passing an american locale, to then replace comas and split on spaces feels very fiddly!

Any chance javascript comes with the equivalent of this? https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html

Reply:
Yes, I don't think there's any native equivalent in browsers.

Just found this, so maybe it's worth using a smaller library https://github.com/you-dont-need/You-Dont-Need-Momentjs

Reply:
Updated to use `dayjs` which has the same `moment` api but is only 2KB https://github.com/iamkun/dayjs